api/config/routes.rb

```diff
@@ -8,124 +8,119 @@
       resources :option_values
-    end
-    resources :option_values
 
-    resources :option_values, only: :index
+      resources :option_values, only: :index
 
-    get '/orders/mine', to: 'orders#mine', as: 'my_orders'
-    get "/orders/current", to: "orders#current", as: "current_order"
+      get '/orders/mine', to: 'orders#mine', as: 'my_orders'
+      get "/orders/current", to: "orders#current", as: "current_order"
 
-    resources :orders, concerns: :order_routes
+      resources :orders, concerns: :order_routes
 
-    resources :zones
-    resources :countries, only: [:index, :show] do
-      resources :states, only: [:index, :show]
-    end
-
-    resources :shipments, only: [:create, :update] do
-      collection do
-        post 'transfer_to_location'
-        post 'transfer_to_shipment'
-        get :mine
+      resources :zones
+      resources :countries, only: [:index, :show] do
+        resources :states, only: [:index, :show]
       end
 
-      member do
-        put :ready
-        put :ship
-        put :add
-        put :remove
-      end
-    end
-    resources :states, only: [:index, :show]
+      resources :shipments, only: [:create, :update] do
+        collection do
+          post 'transfer_to_location'
+          post 'transfer_to_shipment'
+          get :mine
+        end
 
-    resources :taxonomies do
-      member do
-        get :jstree
+        member do
+          put :ready
+          put :ship
+          put :add
+          put :remove
+        end
       end
-      resources :taxons do
+      resources :states, only: [:index, :show]
+
+      resources :taxonomies do
         member do
           get :jstree
         end
+        resources :taxons do
+          member do
+            get :jstree
+          end
+        end
       end
-    end
-
-    resources :taxons, only: [:index]
 
-    resources :inventory_units, only: [:show, :update]
+      resources :taxons, only: [:index]
 
-    resources :users do
-      resources :credit_cards, only: [:index]
-    end
+      resources :inventory_units, only: [:show, :update]
 
-    resources :properties
-    resources :stock_locations do
-      resources :stock_movements
-      resources :stock_items
-    end
+      resources :users do
+        resources :credit_cards, only: [:index]
+      end
 
-    resources :stock_items, only: [:index, :update, :destroy]
-    resources :stores
+      resources :properties
+      resources :stock_locations do
+        resources :stock_movements
+        resources :stock_items
+      end
 
-    put '/classifications', to: 'classifications#update', as: :classifications
-    get '/taxons/products', to: 'taxons#products', as: :taxon_products
-  end
+      resources :stock_items, only: [:index, :update, :destroy]
+      resources :stores
 
-  namespace :api, defaults: { format: 'json' } do
-    concerns :api
-    namespace :v1 do
-      concerns :api
+      put '/classifications', to: 'classifications#update', as: :classifications
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.